docs/README.md

```diff
@@ -0,0 +1,229 @@
+# Indexer API Ddocs
+
+<!-- no toc -->
+- [Endpoints](#endpoints)
+- [Response Format](response-format.md)
+- [Pagination](pagination.md)
+- [Rate Limiting](#rate-limiting)
+
+## Endpoints
+
+#### GET /chains
+
+Returns a list of supported chains and features. `chainId` is the unique identifier for each chain and is used in other endpoints. While chain id returned is a number, endpoints can also accept chain id as hex string (e.g. `0x1e` instead of `30` for Rootstock).
+
+Example response:
+```json
+{
+  "data": [
+    {
+      "chainId": 30,
+      "name": "Rootstock",
+      "features": [
+        "sdex"
+      ]
+    }
+  ],
+  "timestamp": 1630000000
+}
+```
+
+#### GET /tokens
+
+Returns a list of supported tokens and their details. Supports `pagination`. 
+Tokens are ordered by `address`, so native tokens are listed first.
+
+Endpoint data is cached for 1 minute.
+
+`GET /tokens?chainId=30&spam=1`
```

Comment:
There is also another endpoint that should be mentioned: `GET /tokens/:tokenAddress`.